frontend/student/src/App.tsx

```diff
@@ -0,0 +1,9 @@
+// This owns global structural layout (nav, global modals, toasters etc) later.
+// This file NEVER imports from any feature domain directly.
+// TanStack Router's <Outlet /> is where all feature routes get rendered.
+
+import { Outlet } from "@tanstack/react-router";
+
+export default function App() {
+  return <Outlet />;
+}
```

Comment:
Can you please explain why we are not following the template in this aspect, I mean why not making the Outlet component directly in the routes/__route.tsx